racetrack_client/racetrack_client/main.py

```diff
@@ -281,6 +281,28 @@ def _list_installed_plugins(
         list_installed_plugins(remote)
 
 
+@cli_plugin.command('download-installed-plugins')
+def _download_installed_plugins(
+    remote: str = typer.Option(default=None, show_default=False, help="Racetrack server's URL or alias name"),
+    out_dir: str = typer.Option(default='.', show_default=True, help='output directory where to save the ZIP file'),
+    out_filename: str = typer.Option(default='installed_plugins.zip', show_default=True, help='filename of the output ZIP file'),
+):
+    """Downloads all plugins installed on a remote Racetrack server"""
+    download_installed_plugins(remote, out_dir, out_filename)
+
+
+@cli_plugin.command('download-plugin')
```

Comment:
Don't need to repeat `plugin` in my opinion as we're already in plugin context: `racetrack plugin download ...`
```suggestion
@cli_plugin.command('download')
```